### PR TITLE
VAGOV-4094: Using facility sidebar template for outreach

### PR DESCRIPTION
**Pull request by acabouet**
## Description
This refactors some templates to make the facility sidebar more generic so it can be used for the outreach hub. Also removes accordion button styling (plus sign) when menu items have no children per Eric Chiu.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ethanteague — COMMENTED** [review]
How are you getting the outreach menu from drupal without the outreach template that specifies this menu, @acabouet ?

**Comment by acabouet** [comment]
From the graphQL query @ethanteague. I'm just passing the outreach sidebar data from that query to the facility side nav template

**Comment by acabouet** [comment]
I did that because the templates were basically identical minus one piece of functionality for menu items with no children that I added to the facility template. This way we only have to manage to sidebar templates instead of one for each section.

**Comment by ethanteague** [comment]
> I did that because the templates were basically identical minus one piece of functionality for menu items with no children that I added to the facility template. This way we only have to manage to two sidebar templates instead of one for each section.

Awesome.

**Review by ethanteague — APPROVED** [review]
LGTM :+1:

File: src/site/navigation/facility_sidebar_nav.drupal.liquid [hunk 1]
```diff
@@ -1,38 +1,42 @@
-<nav id="va-detailpage-sidebar" data-drupal-sidebar class="va-c-facility-sidebar usa-width-one-fourth va-sidebarnav va-c-facility-sidebar">
```
Comment (jbalboni) [inline src/site/navigation/facility_sidebar_nav.drupal.liquid]: I think there are some styles that use `va-c-facility-sidebar`. Do those need to be removed or updated?

Comment (acabouet) [reply src/site/navigation/facility_sidebar_nav.drupal.liquid]: the class is still there, it was just listed twice. This commit removes the second occurrence.

Comment (jbalboni) [reply src/site/navigation/facility_sidebar_nav.drupal.liquid]: Ha, so it is.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**